.evergreen/run-benchmarks.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+source "${PROJECT_DIRECTORY}/.evergreen/init-node-and-npm-env.sh"
+
+case $1 in
+  "node")
+    export WEB=false
+    npm run check:bench
+    ;;
+  "web")
+    export WEB=true
```

Comment:
Per our slack convo we're going to skip testing the WEB setting for now. 

But for structuring this script, could we just let the evg config set WEB to true/false? just so there isn't multple ways to achieve the same thing